typhon/cloudmask/cloudstatistics.py

```diff
@@ -103,11 +104,11 @@ def iorg(cloudmask, connectivity=1):
     nn = neighbor_distance(cloudmask, connectivity=connectivity)
     nn_sorted = np.sort(nn)
     
-    nncdf = np.array(range(len(neighbor_distance))) / len(neighbor_distance)
+    nncdf = np.array(range(len(nn))) / len(nn)
```

Comment:
Is it correct that nncdf should range from 0 to (1-1/len(nn)), not from 0-1?
Instead of creating a new list object and convert it into a numpy array, using nlinspace is more efficient:
```python
nncdf = np.linspace(0, 1, len(nn), endpoint=False)  # 1 not included
nncdf = np.linspace(0, 1, len(nn))  # 1 included
```